feat: add full screen pop-up [LOGIN-753] (#13)
xsolla/xsolla-login-js-sdk: README.md

<script type="text/javascript">
XL.init({
  projectId: '{Login projectId}',
  callbackUrl: '{callbackUrl}',
  locale: 'en_US',

>   onlyWidgets: true,

Reviewer: ```suggestion
```

Reviewer: @ikukarkin Раз уж из доки убрали, надо и из примеров (в т.ч. из html файлов) убрать

  fields: 'email'
});
</script>
```
Parameter | Description
------------|----
`projectId` | Login ID from Publisher Account. **Required**.
`callbackUrl` | URL to redirect the user to after authentication. Must be identical to **Callback URL** specified in Publisher Account in Login settings. **Required** if there are several Callback URLs.
`locale` | User regional settings.
`fields` | List of parameters required to complete the registration, separated by commas.
`theme` | URL with the widget styles file. If the value is empty, styles uploaded to Publisher Account are used.
`popupBackgroundColor` | Color for background of fullscreen popup window. Accept any CSS legal color values. *'rgba(50, 150, 150, 0.1)'* by default

> `iframeZIndex` | Sets the stack order of fullscreen popup. *1000000* by default.

Reviewer 2: ```suggestion
`iframeZIndex` | Specifies the stack order of the widget. Default is '1000000'.
```

<script type="text/javascript">
XL.init({
  projectId: '{Login projectId}',
  callbackUrl: '{callbackUrl}',
  locale: 'en_US',
  onlyWidgets: true,
  fields: 'email',

>   popupBackgroundColor: 'rgba(50, 150, 150, 0.1)'

Reviewer: Из примера настройку цвета лучше убрать, чтоб бездумно не копировали

});
</script>
```
Parameter | Decsription
------------|----
`projectId` | Login ID from Publisher Account. **Required**.
`callbackUrl` | URL to redirect the user to after authentication. Must be identical to **Callback URL** specified in Publisher Account in Login settings. **Required** if there are several Callback URLs.
`locale` | User regional settings.
`onlyWidgets` | Whether the Login Widget UI is used. *true* by default.
`fields` | List of parameters required to complete the registration, separated by commas.
`theme` | URL with the widget styles file. If the value is empty, styles uploaded to Publisher Account are used.

> `popupBackgroundColor` | Color for background of fullscreen popup window

Reviewer: Нужно добавить в описание, что в качестве значения принимается любое допустимое CSS значние https://www.w3schools.com/cssref/css_colors_legal.asp

> 

Reviewer: Еще нужно добавить описание про `iframeZIndex`

Parameter | Decsription
------------|----
`projectId` | Login ID from Publisher Account. **Required**.
`callbackUrl` | URL to redirect the user to after authentication. Must be identical to **Callback URL** specified in Publisher Account in Login settings. **Required** if there are several Callback URLs.
`locale` | User regional settings.

> `onlyWidgets` | Whether the Login Widget UI is used. *true* by default.

Reviewer 2: ```suggestion
```

`fields` | List of parameters required to complete the registration, separated by commas.
`theme` | URL with the widget styles file. If the value is empty, styles uploaded to Publisher Account are used.

> `popupBackgroundColor` | Color for background of fullscreen popup window. Accept any CSS legal color values. *'rgba(50, 150, 150, 0.1)'* by default

Reviewer 2: ```suggestion
`popupBackgroundColor` | Color for background of fullscreen popup window. Accept any CSS legal color values. *'rgba(50, 150, 150, 0.1)'* by default
```
```suggestion
`popupBackgroundColor` | Color for background of fullscreen popup window. Accept any CSS legal color values. *'rgba(50, 150, 150, 0.1)'* by default
```
```suggestion
`popupBackgroundColor` | Color for background of fullscreen popup window. Accept any CSS legal color values. *'rgba(50, 150, 150, 0.1)'* by default
```
```suggestion
`popupBackgroundColor` | Background color in the fullscreen mode. The value can be in any of the CSS color formats. Default is 'rgba(50, 150, 150, 0.1)'.
```

Reviewer 2: я добавила только одно)

XL.AuthWidget(element_id, options);
</script>
```
Parameter | Description
------------|----
`element_id` | ID of the block containing the Login Widget. **Required**.
`options` | Login Widget block settings. The object consists of the parameters listed below.
`width` | Block width in pixels. Default is 400.
`height` | Block height in pixels. Default is 550.
`route` | Widget start page. Can be: `XL.ROUTES.LOGIN`(by default), `XL.ROUTES.REGISTRATION`, `XL.ROUTES.RECOVER_PASSWORD`, `XL.ROUTES.ALL_SOCIALS`.

> ## Additionally

Reviewer 2: ```suggestion
## Fullscreen mode
```

> Add the button with on click event and call **XL.Show()**. Click on this button will open widget in fullscreen popup window. To close it, click on any area around widget.

Reviewer 2: ```suggestion
To open the widget in the fullscreen mode, add the button with an on-click event to your website and call the **XL.Show()** function. The fullscreen mode is closed by clicking outside of the widget.
```

``` html

> <button onclick="XL.show()">Full screen</button>

Reviewer 2: ```suggestion
<button onclick="XL.show()">Fullscreen widget</button>
```
